contracts/audit-resolve.json

```diff
@@ -1,44 +1,36 @@
 {
     "decisions": {
-        "1094556|@microsoft/api-extractor>@microsoft/api-extractor-model>@rushstack/node-core-library>semver": {
+        "1095367|@microsoft/api-extractor>@microsoft/api-extractor-model>@rushstack/node-core-library>semver": {
```

Comment:
@dnkolegov what's the plan to deal with the rest of the ignored audit issues here?

Reply:
We can't fix them. Ignoring means we can't fix and accept them.

Reply:
Copied context:

> The ignored vulnerabilities are 3rd packages in ganache. We can't update them. We use the latest version of ganache.